package.json

```diff
@@ -53,15 +53,13 @@
     "eventsource": "~2.0.2",
     "fs-extra": "^10.0.0",
     "glob-parent": "^6.0.2",
-    "jest": "^26.6.3",
```

Comment:
We used to need to pin jest, because of an issue with different jest versions breaking different tests being run on our frameworks. However, our tests no longer need this. 

And I won't be able to use the latest version of the vue3-jest library unless we update to jest 27.